tasks/task119_zest_text_modification.json

```diff
@@ -8,7 +8,7 @@
     "Categories": [
         "Text Modification"
     ],
-    "Definition": "Paraphrase the given questions to have different wording. Your paraphrased questions should have the same answer as the original question. Try to change the sentence as much as possible using synonyms and/or rearranging the structure of the sentence. \nThings to avoid: 1. Do not write questions that compare or involve multiple domains, 2. Do not write open-ended or subjective questions. \n\nEmphasis & Caution: 1. Make your questions specific and concrete, 2. The question should have the same type of answer as the original, 3.All the words that are fully capitalized are keywords. For example, in What wars did PRESIDENT initiate as president, the word PRESIDENT is a keyword (which will be replaced with a name of an actual president). Hence, in the paraphrasing this keyword should also be used as fully capitalized.",
+    "Definition": "Paraphrase the given questions to have different wording. Your paraphrased questions should have the same answer as the original question. Try to change the sentence as much as possible using synonyms and/or rearranging the structure of the sentence. \nThings to avoid: 1. Do not write questions that compare or involve multiple domains, 2. Do not write open-ended or subjective questions(e.g. questions that can be answered differently by different people.) \n\nEmphasis & Caution: 1. Make your questions specific and concrete, 2. Your question should have the same type of answer as the original question(e.g. if the answer to the original question is a kind of plant, the answer to your question should also be a kind of plant.), 3.All the words that are fully capitalized are keywords. For example, in What wars did PRESIDENT initiate as president, the word PRESIDENT is a keyword (which will be replaced with the name of an actual president). Hence, in the paraphrasing this keyword should also be used as fully capitalized.",
```

Comment:
Makes sense to me. Maybe @yeganehkordi can double-check it.

Reply:
I checked this again. The same type of answer here means that if a question is extractive, the paraphrased one should be extractive as well. I guess `if the answer to the original question is a kind of plant, the answer to your question should also be a kind of plant` is about having the same answer.